notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>we should probably cover this fetch stuff in the zeroth tutorial rather than here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>